cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -199,6 +201,36 @@ func generateInitramfsMounts() (err error) {
 	return nil
 }
 
+func canInstallAndRun(mst *initramfsMountsState) (bool, error) {
```

Comment:
canInstallAndRunAtOnce is probably clearer? also this probably merits a doc comment